android-components/components/feature/search/src/test/java/mozilla/components/feature/search/middleware/SearchMiddlewareTest.kt

```diff
@@ -1243,6 +1243,50 @@ class SearchMiddlewareTest {
         assertFalse(store.state.search.disabledSearchEngineIds.contains("bing"))
     }
 
+    @Test
+    fun `WHEN restore hidden search engines action THEN hidden engines are added back to bundled engines list`() = runTestOnMain {
+        val metadataStorage = SearchMetadataStorage(testContext, preferences = lazy { FakeSharedPreferences() }) // SearchMetadataStorage(testContext, preferences = lazy { FakeSharedPreferences() })
```

Comment:
```suggestion
        val metadataStorage = SearchMetadataStorage(testContext, preferences = lazy { FakeSharedPreferences() })
```

Unnecessary comment? 🙂